legacy/mbed-greentea/MANIFEST.in

```diff
@@ -0,0 +1,2 @@
+include README.md LICENSE CONTRIBUTING.md
```

Comment:
Perhaps this is not the time to change this, but it feels like these files are not needed in the _delivered pip package_